Samples/VSTO-shared-code-migration/README.md

```diff
@@ -0,0 +1,158 @@
+
+Visual Studio Tools for Office (VSTO) add-ins can only run in Office on Windows. By migrating your code to an Office web add-in, you can expand the reach of your add-in to mobile and online platforms. Migrating the code will involve rewriting it in the JavaScript language, and also updating it to use the Office.js APIs. However, you can save on migration cost by creating a shared library and keeping some of your existing add-in code in its current language.
+
+This sample includes a start folder that accompanies the article TBD... If you want to learn how to go through the steps over creating a shared library for an existing VSTO add-in, follow this article.
+
+And the sample contains a complete folder that contains the completed solution. If you want to explore each section of the completed solution, follow the steps in this readme to learn more.
+
+The sample starts with an Excel VSTO add-in that provides functionality for the user to analyze a cell of text. They can view the unicode values of the text characters. The logic to convert the text to unicode is what will be shared to the web add-in.
+
+![Overview diagram of Excel custom function calling an Azure function that returns a result.](images/azure-function-overview.png)
+
+## Applies to
+
+-  Excel, Outlook, PowerPoint, Word on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+To set up your development environment:
+
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+2. Install the following workloads:
+    a. ASP.NET and web development
+    b. .NET Core cross-platform development
+    c. Office/SharePoint development
+    d. Visual Studio Tools for Office (VSTO) Note that this is an Individual component.
+
+You will also need the following:
+
+- An Office 365 account. You can join the [Office 365 Developer Program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Shared library VSTO migration | David Chesnut (**Microsoft**)
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0  | December 2, 2019 | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+## Analyze your VSTO add-in
+
+The first step to migration is to analyze the code in your VSTO add-in. You'll want to identify each section of code as following into one of the following three categories.
+
+1. UI code that interacts with the user, such as task pane code that uses a WinForm.
+2. Document code that interacts with the document through the Office APIs, such as updating Excel range values.
+3. Business logic, such as algorithms or helper functions that can work independently from the UI or document.
```

Comment:
I think there should be another comma after "helper functions"

Reply:
fixed